tools/update.hpp

```diff
@@ -110,23 +110,36 @@ namespace antler {
       update_project(CLI::App& app) {
          path = system::fs::current_path().string();
          subcommand = app.add_subcommand("update", "Update an app, dependency, library or test to your project.");
-         subcommand->add_option("-p, path", path, "This must be the path to the `project.yml` or the path containing it.")->default_val(".");
+         subcommand->add_option("-p, path", path, "Path containing the project's yaml file.");
 
-         app_subcommand = subcommand->add_subcommand("app", "Remove app from the project.");
+         app_subcommand = subcommand->add_subcommand("app", "Update an app in the project.");
+         app_subcommand->footer(std::string(R"(Examples:)")
+               + "\n\t" + app.get_name() +R"( update app MyApp C++ \\-O2)"
+               + "\n\t" + app.get_name() +R"( ./path-to-project/ update app -n MyApp --comp -O2)");
+         app_subcommand->add_option("-p", path, "Path containing the project's yaml file.");
          app_subcommand->add_option("-n, name", obj_name, "The name of the app to remove.")->required();
          app_subcommand->add_option("-l, language", lang, "The language of the app.");
          app_subcommand->add_option("--comp, compile_options", copts, "The compile options used to build the app.")
             ->transform(escape_transform);
          app_subcommand->add_option("--link, link_options", lopts, "The link options used to build the app.")
             ->transform(escape_transform);
 
-         lib_subcommand = subcommand->add_subcommand("lib", "Remove lib from the project.");
+         lib_subcommand = subcommand->add_subcommand("lib", "Update a lib in the project.");
+         lib_subcommand->footer(std::string(R"(Examples:)")
+               + "\n\t" + app.get_name() +R"( update lib MyLib C++ \\-O2 "\-s")"

```

Comment:
When running this command there is a second core dump: https://github.com/AntelopeIO/antler-proj/issues/38#issuecomment-1508045945

Reply:
Corrected in c4c355b